### PR TITLE
bids-qsiprep.json creation

### PR DESCRIPTION
**Pull request by dpark6060**
0.0.0_0.12.2

### CODE REVIEW

File: gears/flywheel/bids-qsiprep.json [hunk 1]
```diff
@@ -0,0 +1,411 @@
+{
+  "name": "bids-qsiprep",
+  "label": "BIDS qsiprep",
+  "description": "BIDS qsiprep 0.0.0_0.12.2 \n qsiprep configures pipelines for processing diffusion-weighted MRI (dMRI) data. The main features of this software are\n\nA BIDS-app approach to preprocessing nearly all kinds of modern diffusion MRI data.\n\nAutomatically generated preprocessing pipelines that correctly group, distortion correct, motion correct, denoise, coregister and resample your scans, producing visual reports and QC metrics.\n\nA system for running state-of-the-art reconstruction pipelines that include algorithms from Dipy, MRTrix, DSI Studio and others.\n\nA novel motion correction algorithm that works on DSI and random q-space sampling schemes",
```
Comment (lmperry) [inline gears/flywheel/bids-qsiprep.json:4]: Are the "\n" interpreted correctly by the UI, or will those be rendered literally?

Comment (dpark6060) [reply gears/flywheel/bids-qsiprep.json:4]: Good catch.  They render correctly for input/config option descriptions, so I assumed they would be here.  However I have verified that they are just ignored - not a line break, but not rendered at all.  

Would actually be nice if they did render correctly, though!

Comment (AndysWorth) [reply gears/flywheel/bids-qsiprep.json:4]: I uploaded it to ga.  They appear to be ignored.

**Review by AndysWorth — APPROVED** [review]
LGTM

**Pull request merged**